README.md

```diff
@@ -5,6 +5,7 @@
 [![Unit Tests][ci-img]][ci]
 [![Coverage Status][cov-img]][cov]
 [![FOSSA Status][fossa-img]][ci]
+[![Artifact Hub][artifacthub-img]
```

Comment:
do we want to link somewhere, like https://artifacthub.io/packages/helm/jaegertracing/jaeger ?

Reply:
If you'd like I can add that in, I'm just checking boxes really for CLOMonitor. I am not certain how used artifact hub actually is....